pkgs/applications/networking/circumflex/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, less, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "circumflex";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    repo = "circumflex";
+    owner = "bensadeh";
+    rev = "2.6";
+    sha256 = "sha256-pcY2PXiOazKAi8mAAbmftXDae01fcUw/u9JPOHQVclI=";
+  };
+
+  subPackage=".";
+  vendorHash = "sha256-rF1Hu4Pf9AF2MTx4GAPmzSn0M38uTxPS1bsAkO23SdI=";
+
+  buildInputs = [ less ];
```

Comment:
Does this really need to be in `buildInputs`?

Reply:
If I understood it correctly buildInputs are the runtime requirements in a sense? If not then I'll remove it, if yes then I found the following in the application I'm packaging (they seem to use `less` behind the scenes)

https://github.com/bensadeh/circumflex/blob/4af60b5e88c61ebd265bcb2353a61aac8900afbf/cli/cli.go#L32

Reply:
Not quite. `buildInputs` will be available in the build sandbox, but are not automatically made available at runtime - anything used at runtime needs to refer to the full store path. It looks like the application expects to find `less` in PATH (and also `clear`, which is part of `ncurses`), so it probably needs to be wrapped using [`wrapProgram`](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper).

Reply:
hmm I see. I fixed it as you suggested, I used `wrapProgram` with `less` and `ncurses`.